.travis.yml

```diff
@@ -6,30 +6,81 @@
 # http://www.eclipse.org/legal/epl-v20.html
 #
 # SPDX-License-Identifier: EPL-2.0
-sudo: false
+sudo: true
 language: java
-cache:
-  apt: true
-  directories:
-    - $HOME/.m2
-before_cache:
-  - rm -rf $HOME/.m2/repository/org/eclipse/sw360/antenna/
-jdk:
-  - openjdk8
-  # - openjdk10
-  # - openjdk11
-  # - oraclejdk8
-  # - oraclejdk9
-  # - oraclejdk11
+addons:
+  apt:
+    packages:
+      - git
+# cache:
+#   apt: true
+#   directories:
+#     - $HOME/.m2
+# before_cache:
+#   - rm -rf $HOME/.m2/repository/org/eclipse/sw360/antenna/
+jdk: openjdk8
+
+before_install:
+  - sudo apt-get update -qq
+  - git clone --branch sw360-3.3.0-M1 --depth 1 https://github.com/eclipse/sw360 /tmp/sw360
+  - sudo /tmp/sw360/scripts/install-thrift.sh --no-cleanup
+  - pushd /tmp/sw360
+  - mvn install -pl backend/src/src-licenseinfo,libraries/lib-datahandler -am -DskipTests
+  - popd
+  - wget -N -O /tmp/install-jdk.sh https://raw.githubusercontent.com/sormuras/bach/ab648ed36892439f49a827cdf18c25e17c8095db/install-jdk.sh
+  - chmod +x /tmp/install-jdk.sh
+install:
+  - mvn install -DskipTests=true -B
 
 jobs:
   include:
-    - name: mvn test
-      install:
-        - mvn clean verify -P '!build-assembly' -DskipTests=true -V -B
-        - mvn install -P '!build-assembly' -DskipTests=true -B
+    # compile and execute with the same JDK
+    - &mvn-test
+      name: mvn test
+      script:
+        - mvn test -B
+        - cd example-projects/example-project
+        - mvn package
```

Comment:
Use `mvn clean test` and `mvn clean package` to isolate the different test cases?

Reply:
mvn package is ran in a subdirectory, which has not yet seen any interaction